contrib/pa-pass

```diff
@@ -47,52 +32,12 @@ usage() {
 main() {
     basedir="${XDG_DATA_HOME:=$HOME/.local/share}/pa"
     : "${PA_DIR:=$basedir/passwords}"
-    identities_file="$basedir/identities"
     recipients_file="$basedir/recipients"
 
-    mkdir -p "$basedir" "$PA_DIR" ||
-        die "Couldn't create pa directories"
-
-    # Ensure that globbing is disabled
-    # to avoid insecurities with word-splitting.
-    set -f
-
-    command -v age >/dev/null 2>&1 ||
-        die "age not found, install per https://github.com/FiloSottile/age"
-
-    command -v age-keygen >/dev/null 2>&1 ||
-        die "age-keygen not found, install per https://github.com/FiloSottile/age"
-
-    glob "$1" '[m]*' && [ -z "$2" ] &&
-	die "Missing [PASSWORD_STORE_DIR] argument"
-
-    # Restrict permissions of any new files to
-    # only the current user.
-    umask 077
-
-    # First, copy any existing identities files from the old
-    # storage location to the new one for backwards compat.
-    # Then, attempt key generation.
-    [ -f "$identities_file" ] ||
-        cp ~/.age/key.txt "$identities_file" 2>/dev/null ||
-        age-keygen -o "$identities_file" 2>/dev/null
-
-    [ -f "$recipients_file" ] ||
-        age-keygen -y -o "$recipients_file" "$identities_file" 2>/dev/null
-
-    # Ensure that we leave the terminal in a usable
-    # state on exit or Ctrl+C.
-    [ -t 1 ] && trap 'stty echo icanon' INT EXIT
-
     case $1 in
-    m*) pw_migrate "$2" ;;
+    m*) pw_migrate "${2:-$HOME/.password-store}" ;;
```

Comment:
Using the `$PASSWORD_STORE_DIR` would be also a nice improvement. but in another PR :eyes:  ;)

Reply:
yeah, but then it kind of defeats the purpose of having it as a positional argument, doesn't it? preferably there should be just one way to configure this value